testing/describes.js

```diff
@@ -739,7 +738,6 @@ class AmpFixture {
       // Notice that ampdoc's themselves install runtime styles in shadow roots.
       // Thus, not changes needed here.
     }
-    maybeTrackImpression(self);
```

Comment:
Curious how this is leaking global state? And can we avoid that by calling `resetTrackImpressionPromiseForTesting()` in `teardown()`?